particles/Tutorial/Kotlin/5_Collections/BUILD

```diff
@@ -9,6 +9,7 @@ licenses(["notice"])
 arcs_kt_schema(
     name = "collections_schemas",
     srcs = ["Collections.arcs"],
+    deps = ["CollectionsSchemas.arcs"],
```

Comment:
I feel this is the wrong way to do this, the deps attribute should reference other arcs_kt_schema rules.

Reply:
Yes, I proposed the same thing: https://docs.google.com/document/d/171O7uJbGHD_ZWSaTcGDSALzr_waRKzd2fz7bJ0KwcBY/edit?disco=AAAAI2ndggY

I will file a bug for this (as I promised Alex) and will try to find someone to revamp our build rules with this and some other improvements.

Reply:
@cromwellian How about it references an arcs_manifest? 

The trouble with referencing an `arcs_kt_schema` is that we only generate entites from particle connections, of which this file has none.